docs/source/examples/learning-depolarizing-noise.myst

```diff
@@ -0,0 +1,221 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name:  python3
+---
+
+# Learning quasiprobability representations with a depolarizing noise model
+
+In this example, we demonstrate the workflow of learning quasiprobability representations of a `CNOT` gate with a depolarizing noise model,
+from Clifford circuit data.
+The depolarizing noise model is parameterized by the noise strength, `epsilon`.
+The resulting quasiprobability representations of the `CNOT` gate are then used to obtain an error-mitigated expectation value with Mitiq's
+{ref}`probabilistic error cancellation module <guide/pec/pec>`. 
+For a more in-depth description of the learning function used in this example, see the section on {func}`.learn_depolarizing_noise_parameter`
+in the API-doc.
+The learning-based PEC workflow was inspired by the procedure described in *Strikis et al. PRX Quantum (2021)* {cite}`Strikis_2021_PRXQuantum`.
+
+```{code-cell} ipython3
+import numpy as np
+from matplotlib import pyplot as plt
+from cirq import (
+    CXPowGate,
+    DepolarizingChannel,
+    Rx,
+    Rz,
+    I,
+    X,
+    Y,
+    Z,
+    LineQubit,
+    Circuit,
+    ops,
+    unitary,
+    InsertStrategy,
+)
+from mitiq import Executor, Observable, PauliString
+from mitiq.interface.mitiq_cirq import compute_density_matrix
+from mitiq.cdr import generate_training_circuits
+from mitiq.cdr._testing import random_x_z_cnot_circuit
+from mitiq.pec import execute_with_pec
+from mitiq.pec.representations.depolarizing import (
+    represent_operation_with_local_depolarizing_noise,
+)
+from mitiq.pec.representations.learning import (
+    biased_noise_loss_function,
+    learn_depolarizing_noise_parameter,
+)
```

Comment:
Please check if we need all the imports. E.g. gates

Reply:
Thanks for the reminder! Just pushed a new commit with cleaned-up imports.